frontend/src/components/toolbar/problem-metadata/problem-metadata.component.tsx

```diff
@@ -61,44 +61,55 @@ export function ProblemMetadataComponent() {
     setSnackbarOpen(true);
   }
 
-  const handleSubmit = async (event) => {
-    event.preventDefault();
+  function handleNumberOfNursesChange() {
+    let { nurseCount } = schedule?.employee_info || {};
 
-    let { nurseCount, babysitterCount } = schedule?.employee_info || {};
-
-    if (!nurseCount || !babysitterCount) {
-      showSnackbar("Wczytany harmonogram nie zawiera ilości pielęgniarek lub opiekunek");
+    if (!nurseCount) {
       return;
     }
 
     if (nurseCount > numberOfNurses) {
       setNumberOfNurses(nurseCount);
       showSnackbar("Wprowadzono mniej pielęgniarek niż w harmonogramie");
     }
+  }
+
+  function handleNumberOfSittersChange() {
+    let { babysitterCount } = schedule?.employee_info || {};
```

Comment:
Tu w sumie można nie używać `let { babysitterCount } = schedule?.employee_info || {};`, a po prostu napisać `babysitterCount = schedule.employee_info.babysitter_count`


Ta składnia jest bardzo przydatna, kiedy chcemy dostać kilka różnych wartości z obiektu i przypisać ich do zmiennych, bo zamiast tego żeby pisać kilka linijkowy kod w stylu
```
let a = obj['a']
let b = obj['b']
....
```
można napisać 
```
let {a,b } = obj
```
Co moim zdaniem jest czytelniejsze. 
Pisałem ci wtedy o tym ponieważ wtedy potrzeba było dostać dwie zmienne z obiektu, a to pozwalało skrócić ten zapis co mieliśmy